cmd/dosa/util.go

```diff
@@ -114,16 +117,35 @@ func strToFieldValue(t dosa.Type, s string) (dosa.FieldValue, error) {
 	}
 }
 
-func printResult(res []map[string]dosa.FieldValue) {
-	if len(res) == 0 {
-		fmt.Println("Error: not found")
-		return
+func printResults(results []map[string]dosa.FieldValue) error {
+	const padding = 3
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, padding, ' ', tabwriter.Debug)
+	if len(results) == 0 {
+		return errors.New("Empty results")
 	}
-	fmt.Println("Results:")
-	for _, fieldValues := range res {
-		for field, value := range fieldValues {
-			fmt.Printf("%s: %v\n", field, reflect.Indirect(reflect.ValueOf(value)))
+	i := 0
+	width := len(results[0])
+	keys := make([]string, width)
+	for k := range results[0] {
+		keys[i] = k
+		i++
+	}
+	sort.Strings(keys)
+	_, err := fmt.Fprintln(w, strings.Join(keys, "\t"))
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	values := make([]string, width)
+	for _, result := range results {
+		i := 0
+		for _, key := range keys {
```

Comment:
nit: here maybe we can write like
'for idx, result := range results'
and get rid of i := 0 and i++?